### PR TITLE
Creating form

### PR DESCRIPTION
**Pull request by huzaifakhan0308**
Made CSS style changes and also add some changes in README file

### CODE REVIEW

**Review by tugsem — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌ 

Hi @huzaifakhan0308 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have the correct github flow. I suggest you to change the base branch using this Github doc -> [Changing the base branch of a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)
- Also, please don't open different pr's for a feature. Use the same pr to ask for a review which have been reviewed before.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huzaifakhan0308** [comment]
> ## Invalid Code Review Request ❌
> Hi @huzaifakhan0308 ,
> 
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> You have submitted a project, that doesn't have the correct github flow. I suggest you to change the base branch using this Github doc -> [Changing the base branch of a pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)
> 
> * Also, please don't open different pr's for a feature. Use the same pr to ask for a review which have been reviewed before.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

Now, what should i do with the 2nd branch (creating-form) . Should I delete it and  make changes in the 1st branch (main-changes).
These branches are already  with the base branch (personal-portfolio)
and if you told me something else, please clarify I didn't understand your point

**Comment by huzaifakhan0308** [comment]
Now, what should i do with the 2nd branch (creating-form) . Should I delete
it and make changes in the 1st branch (main-changes).
These branches are already with the base branch (personal-portfolio)
and if you told me something else, please clarify I didn't understand your
point

File: index.css [hunk 1]
```diff
@@ -371,3 +371,116 @@ main {
...
+  margin-top: 15px;
+  border-radius: 8px;
+  border: none;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 600;
+  font-size: 17px;
+  cursor: pointer;
+  background-color: white;
+}
```
Comment (Qoosim) [inline index.css:458]: - [ ] I would like recommend you to consider adding more padding to your `Get in Touch` button in order to be inline with the design on the figma template.

### Your Design
your-btn

### Figma Design
figma-btn

File: index.css [hunk 2]
```diff
@@ -371,3 +371,116 @@ main {
...
+  font-family: "Poppins", sans-serif;
+}
+
+.form-container {
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin-top: 5vh;
+}
```
Comment (Qoosim) [inline index.css:425]: - [ ] In order to get close replica of the guiding template, you need to produce almost the same design like we have on the figma design. Kindly compare your design with the figma design and work your design alongside figma design.

### Your Design
your-form

### Figma Design
figma-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @huzaifakhan0308, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @huzaifakhan0308, 

I am Favour 👩🏽 , your reviewer for this Pull Request. 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

happy

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**